contracts/base/GeneralMiddleware.sol

```diff
@@ -37,6 +39,7 @@ contract GeneralMiddleware is IbcMwUser, IbcMiddleware, IbcMwEventsEmitter {
         bytes calldata appData,
         uint64 timeoutTimestamp
     ) external override {
+        emit UCHPacketSent(msg.sender, destPortAddr);
```

Comment:
@alfredo-stonk pretty straightforward; but this is the implementation re: our discussion a couple of weeks ago